# Fix includes

diff -N -r -U 3 libffado-2.0.1-orig/src/bebob/bebob_dl_mgr.cpp libffado-2.0.1/src/bebob/bebob_dl_mgr.cpp
--- libffado-2.0.1-orig/src/bebob/bebob_dl_mgr.cpp	2009-05-17 16:42:38.000000000 +0300
+++ libffado-2.0.1/src/bebob/bebob_dl_mgr.cpp	2010-09-04 21:20:51.000000000 +0300
@@ -35,6 +35,7 @@
 
 #include <cstdio>
 #include <cstring>
+#include <memory>
 
 namespace BeBoB {
     enum {
diff -N -r -U 3 libffado-2.0.1-orig/src/ffadodevice.h libffado-2.0.1/src/ffadodevice.h
--- libffado-2.0.1-orig/src/ffadodevice.h	2008-11-23 13:25:32.000000000 +0200
+++ libffado-2.0.1/src/ffadodevice.h	2010-09-04 21:19:52.000000000 +0300
@@ -34,6 +34,7 @@
 
 #include <vector>
 #include <string>
+#include <memory>
 
 class DeviceManager;
 class ConfigRom;
diff -N -r -U 3 libffado-2.0.1-orig/src/libutil/serialize_expat.cpp libffado-2.0.1/src/libutil/serialize_expat.cpp
--- libffado-2.0.1-orig/src/libutil/serialize_expat.cpp	2009-11-22 19:05:16.000000000 +0200
+++ libffado-2.0.1/src/libutil/serialize_expat.cpp	2010-09-04 21:24:46.000000000 +0300
@@ -25,6 +25,8 @@
 
 #include "serialize.h"
 
+#include <cstdlib>
+
 using namespace std;
 
 
